core/shared/mem-alloc/ems/ems_kfc.c

```diff
@@ -132,10 +132,6 @@ gc_destroy_with_pool(gc_handle_t handle)
         && (hmu_t *)((char *)cur + hmu_get_size(cur)) != end) {
         os_printf("Memory leak detected:\n");
         gci_dump(heap);
-#if WASM_ENABLE_SPEC_TEST != 0
```

Comment:
Can we add a config `BH_EXIT_ON_GC_VERIFY_ERROR` and exit(-1) here when it is enabled? This will be more friendly to instrumentation test framework

Reply:
Yes, will the return value -1 be gotten by the runtime embedder?

Reply:
> Yes, will the return value -1 be gotten by the runtime embedder?

the -1 will be the exit code of the process and will be gotten by another process (instrument daemon server)

Reply:
Add the exit(-1) in wasm_runtime_memory_destroy(), wasm_memory.c, for global heap only. As wasm app heap also uses ems gc, if it has memory leak, it will exit firstly, the global heap memory leak check will be ignored.